xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/xwiki.properties.vm

```diff
@@ -933,6 +933,23 @@ distribution.automaticStartOnWiki=$xwikiPropertiesAutomaticStartOnWiki
 #-# with programming rights, while this configuration affect any attachment.
 # attachment.download.forceDownload=
 
+#-# [Since 14.10RC1]
+#-# Define the list of allowed attachment mimetypes. By default, this list is empty, but you can specify a 
+#-# comma-separated list of allowed mimetypes. Note that the mimetypes are interpreted as regular expressions. Once this
+#-# list is not empty, any attachment with an unlisted mimetype will be rejected.
+#-# For instance, if you want to only accept plain text and any kind of images, you can set the list to:
+#-# text/plain,image/.*
+# attachment.mimetype.allowList=
```

Comment:
The naming is not very consistent with existing properties, `attachment.download.whitelist` too is related to mime types.

Since it's about what you are allowed to upload, wouldn't it make more sense to name it something like `attachment.upload.allowList` (I assume the move from `whitelist` to `allowList` is for "using black vs white is not a great idea" reasons) ?

Reply:
> (I assume the move from whitelist to allowList is for "using black vs white is not a great idea" reasons)

Exactly :)

I'm fine with `attachment.upload.allowList`, I'll update accordingly (with some delay to let other answer as well).

Reply:
done